RevenueCatUI/Templates/V2/ViewModelHelpers/PaywallComponentViewModel.swift

```diff
@@ -28,7 +28,6 @@ enum PaywallComponentViewModel {
     case button(ButtonComponentViewModel)
     case package(PackageComponentViewModel)
     case purchaseButton(PurchaseButtonComponentViewModel)
-    case stickyFooter(StickyFooterComponentViewModel)
```

Comment:
Same as other spot... removed here because sticky footer is never going to be in the normal view stack tree thingy